openproblems/tasks/_batch_integration/_common/datasets/pancreas.py

```diff
@@ -35,6 +35,16 @@ def pancreas_batch(test: bool = False, min_celltype_count: Optional[int] = None)
     )
     adata.obsm["X_uni_pca"] = adata.obsm["X_pca"]
 
+    hvg_unint = sc.pp.highly_variable_genes(
+        adata,
+        n_top_genes=2000,
+        layer="log_normalized",
+        flavor="cell_ranger",
+        batch_key="batch",
+        inplace=False,
+    )
+    adata.uns["hvg_unint"] = list(hvg_unint[hvg_unint.highly_variable].index)
```

Comment:
Might be good to roll up lines 38-46 into a utils function to avoid repeated/copypasted code